CONTRIBUTING.rst

```diff
@@ -50,6 +50,17 @@ that black makes to your code after those changes are applied.
 
     $ pre-commit install
 
+Running tests
+^^^^^^^^^^^^^
+To run all of the tests (from the root directory of the repo):
+
+    $ pytest
```

Comment:
i'm not sure if i just made a suggestion or a commit. i need to figure out how to create a suggestion (what i was trying to do.) it looks like i committed it directly tho.